docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,188 @@
+.. _complex_numbers-doc:
+
+Complex Numbers
+===============
+
+Complex Numbers are numbers that can be expressed in the form :math:`a + bi`, where a and b are real numbers,
+and *i* is a solution of the equation :math:`x^2 = −1`. Complex numbers frequently occur in mathematics and
+engineering, especially in signal processing. The aim of introducing tensors of complex dtypes is to provide
+a more natural user experience for users and libraries (eg. TorchAudio) that currently work around the
+lack of complex tensors by using Float Tensors with shape :math:`(..., 2)` where the last dimension contains
+the real and imaginary values.
+
+We support many functions for Complex Tensors eg. :func:`torch.svd`, :func:`torch.qr` etc. Operations on complex tensors are likely to be
+faster than operations on float tensors mimicking them. Operations involving complex numbers in PyTorch are
+optimized to use vectorized assembly instructions and specialized kernels (e.g. LAPACK, CuBlas). Thus using
+functions for complex tensors will provide performance benefits as opposed to users defining their own functions.
+
+.. warning ::
+     Complex Tensors is a beta feature and subject to change.
+
+Creating Complex Tensors
+----------------------
+
+We support two complex dtypes: `torch.cfloat` and `torch.cdouble`
+
+::
+
+     >>> x = torch.randn(2,2, dtype=torch.cfloat)
+     >>> x
+     tensor([[-0.4621-0.0303j, -0.2438-0.5874j],
+          [ 0.7706+0.1421j,  1.2110+0.1918j]])
+
+.. note::
+
+     The default dtype for complex tensors is determined by the default floating point dtype.
+     If the default floating point dtype is torch.float64 then complex numbers are inferred to
+     have a dtype of torch.complex128, otherwise they are assumed to have a dtype of torch.complex64.
+
+The following factory functions can be used to create complex tensors:
+
+- :func:`torch.tensor`
+- :func:`torch.as_tensor`
+- :func:`torch.empty`
+- :func:`torch.rand`
+- :func:`torch.randn`
+- :func:`torch.as_strided`
+- :func:`torch.from_numpy`
+- :func:`torch.zeros`
+- :func:`torch.ones`
+- :func:`torch.full`
+- :func:`torch.eye`
+
+Transition from the old representation
+--------------------------------------
+
+Users who currently worked around the lack of complex tensors with real tensors of shape `(..., 2)`
+can easily to switch using the complex tensors in their code using :func:`torch.view_as_complex` and
+- :func:`torch.view_as_real` view functions:
```

Comment:
I'm not sure that just saying "view functions" makes it clear to the user that these are views. Maybe you want to insist on it a bit more as it is fairly important.

Reply:
`...:func:`torch.view_as_complex` and - :func:`torch.view_as_real` functions which involve zero copy.` does that look better?

Reply:
Or add a new setence: Note that these functions don't perform any copy and return a view of the input Tensor.